go.work

```diff
@@ -0,0 +1,129 @@
+	comp/core/config
+	comp/core/flare/builder
+	comp/core/flare/types
+	comp/core/hostname/hostnameinterface
+	comp/core/log/def
+	comp/core/log/impl
+	comp/core/log/impl-trace
+	comp/core/log/mock
+	comp/core/secrets
+	comp/core/status
+	comp/core/status/statusimpl
+	comp/core/tagger/tags
+	comp/core/tagger/types
+	comp/core/tagger/utils
+	comp/core/telemetry
+	comp/def
+	comp/forwarder/defaultforwarder
+	comp/forwarder/orchestrator/orchestratorinterface
+	comp/logs/agent/config
+	comp/netflow/payload
+	comp/otelcol/collector-contrib/def
+	comp/otelcol/collector-contrib/impl
+	comp/otelcol/converter/def
+	comp/otelcol/converter/impl
+	comp/otelcol/ddflareextension/def
+	comp/otelcol/ddflareextension/impl
+	comp/otelcol/logsagentpipeline
+	comp/otelcol/logsagentpipeline/logsagentpipelineimpl
+	comp/otelcol/otlp/components/exporter/datadogexporter
+	comp/otelcol/otlp/components/exporter/logsagentexporter
+	comp/otelcol/otlp/components/exporter/serializerexporter
+	comp/otelcol/otlp/components/metricsclient
+	comp/otelcol/otlp/components/processor/infraattributesprocessor
+	comp/otelcol/otlp/components/statsprocessor
+	comp/otelcol/otlp/testutil
+	comp/serializer/compression
+	comp/trace/agent/def
+	comp/trace/compression/def
+	comp/trace/compression/impl-gzip
+	comp/trace/compression/impl-zstd
+	internal/tools
+	internal/tools/independent-lint
+	internal/tools/modformatter
+	internal/tools/modparser
+	internal/tools/proto
+	pkg/aggregator/ckey
+	pkg/api
+	pkg/collector/check/defaults
+	pkg/config/env
+	pkg/config/mock
+	pkg/config/model
+	pkg/config/nodetreemodel
+	pkg/config/remote
+	pkg/config/setup
+	pkg/config/structure
+	pkg/config/teeconfig
+	pkg/config/utils
+	pkg/errors
+	pkg/gohai
+	pkg/linters/components/pkgconfigusage
+	pkg/logs/auditor
+	pkg/logs/client
+	pkg/logs/diagnostic
+	pkg/logs/message
+	pkg/logs/metrics
+	pkg/logs/pipeline
+	pkg/logs/processor
+	pkg/logs/sds
+	pkg/logs/sender
+	pkg/logs/sources
+	pkg/logs/status/statusinterface
+	pkg/logs/status/utils
+	pkg/logs/util/testutils
+	pkg/metrics
+	pkg/networkdevice/profile
+	pkg/obfuscate
+	pkg/orchestrator/model
+	pkg/process/util/api
+	pkg/proto
+	pkg/remoteconfig/state
+	pkg/security/secl
+	pkg/security/seclwin
+	pkg/serializer
+	pkg/status/health
+	pkg/tagger/types
+	pkg/tagset
+	pkg/telemetry
+	pkg/trace
+	pkg/trace/stats/oteltest
+	pkg/util/backoff
+	pkg/util/buf
+	pkg/util/cache
+	pkg/util/cgroups
+	pkg/util/common
+	pkg/util/containers/image
+	pkg/util/defaultpaths
+	pkg/util/executable
+	pkg/util/filesystem
+	pkg/util/flavor
+	pkg/util/fxutil
+	pkg/util/grpc
+	pkg/util/hostname/validate
+	pkg/util/http
+	pkg/util/json
+	pkg/util/log
+	pkg/util/log/setup
+	pkg/util/optional
+	pkg/util/pointer
+	pkg/util/scrubber
+	pkg/util/sort
+	pkg/util/startstop
+	pkg/util/statstracker
+	pkg/util/system
+	pkg/util/system/socket
+	pkg/util/testutil
+	pkg/util/utilizationtracker
+	pkg/util/uuid
+	pkg/util/winutil
+	pkg/version
+	test/fakeintake
+	test/new-e2e
+	test/otel
+)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
🎉